cms/templates/asset_index.html

```diff
@@ -35,32 +35,31 @@
 
 <div class="wrapper-mast wrapper">
     <header class="mast has-actions has-subtitle">
-        <h1 class="page-header">
+        <h3 class="page-header">
             <small class="subtitle">${_("Content")}</small>
-            <span class="sr">&gt; </span>${_("Files & Uploads")}
-        </h1>
+            <span class="sr">- </span>${_("Files & Uploads")}
+        </h3>
 
-        <nav class="nav-actions" aria-label="${_('Page Actions')}">
-            <h3 class="sr">${_("Page Actions")}</h3>
+        <div class="nav-actions" aria-label="${_('Page Actions')}">
             <ul>
                 <li class="nav-item">
                     <a href="#" class="button upload-button new-button"><span class="icon fa fa-plus" aria-hidden="true"></span> ${_("Upload New File")}</a>
                 </li>
             </ul>
-        </nav>
+        </div>
     </header>
 </div>
 
 <div class="wrapper-content wrapper">
-    <section class="content">
-        <article class="content-primary" role="main">
-            <div class="wrapper-assets" />
+    <div class="content">
+        <div class="content-primary">
+            <div class="wrapper-assets"> </div>
             <div class="ui-loading">
                 <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
             </div>
-        </article>
+        </div>
 
-        <aside class="content-supplementary" role="complementary">
+        <aside class="content-supplementary" role="complementary" aria-label="Help adding Files and Uploads">
```

Comment:
You must mark strings that a user will encounter for translation.

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html